storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/operations/delete/PreparedDeleteObjectTest.java

```diff
@@ -131,7 +132,7 @@ public void shouldThrowExceptionIfNoTypeMappingWasFoundWithoutAffectingContentPr
 
             testSubscriber.awaitTerminalEvent();
             testSubscriber.assertNoValues();
-            testSubscriber.assertError(IllegalStateException.class);
+            testSubscriber.assertError(StorIOException.class);
```

Comment:
Please add assertion for exception cause type and message, we must be sure that it's really expected exception.